presto-main/src/main/java/io/prestosql/security/InjectedConnectorAccessControl.java

```diff
@@ -0,0 +1,365 @@
+    }
+
+    @Override
+    public void checkCanDropRole(ConnectorSecurityContext context, String role)
+    {
+        checkArgument(context == null, "context must be null");
+        accessControl.checkCanDropRole(securityContext, role, catalogName);
+    }
+
+    @Override
+    public void checkCanGrantRoles(ConnectorSecurityContext context, Set<String> roles, Set<PrestoPrincipal> grantees, boolean adminOption, Optional<PrestoPrincipal> grantor, String catalogName)
+    {
+        checkArgument(context == null, "context must be null");
+        accessControl.checkCanGrantRoles(securityContext, roles, grantees, adminOption, grantor, catalogName);
+    }
+
+    @Override
+    public void checkCanRevokeRoles(ConnectorSecurityContext context, Set<String> roles, Set<PrestoPrincipal> grantees, boolean adminOption, Optional<PrestoPrincipal> grantor, String catalogName)
+    {
+        checkArgument(context == null, "context must be null");
+        accessControl.checkCanRevokeRoles(securityContext, roles, grantees, adminOption, grantor, catalogName);
+    }
+
+    @Override
+    public void checkCanSetRole(ConnectorSecurityContext context, String role, String catalogName)
+    {
+        checkArgument(context == null, "context must be null");
+        accessControl.checkCanSetRole(securityContext, role, catalogName);
+    }
+
+    @Override
+    public void checkCanShowRoleAuthorizationDescriptors(ConnectorSecurityContext context, String catalogName)
+    {
+        checkArgument(context == null, "context must be null");
+        accessControl.checkCanShowRoleAuthorizationDescriptors(securityContext, catalogName);
+    }
+
+    @Override
+    public void checkCanShowRoles(ConnectorSecurityContext context, String catalogName)
+    {
+        checkArgument(context == null, "context must be null");
+        accessControl.checkCanShowRoles(securityContext, catalogName);
+    }
+
+    @Override
+    public void checkCanShowCurrentRoles(ConnectorSecurityContext context, String catalogName)
+    {
+        checkArgument(context == null, "context must be null");
+        accessControl.checkCanShowCurrentRoles(securityContext, catalogName);
+    }
+
+    @Override
+    public void checkCanShowRoleGrants(ConnectorSecurityContext context, String catalogName)
+    {
+        checkArgument(context == null, "context must be null");
+        accessControl.checkCanShowRoleGrants(securityContext, catalogName);
+    }
+
+    @Override
+    public void checkCanExecuteProcedure(ConnectorSecurityContext context, SchemaRoutineName procedure)
+    {
+        checkArgument(context == null, "context must be null");
+        accessControl.checkCanExecuteProcedure(securityContext, new QualifiedObjectName(catalogName, procedure.getSchemaName(), procedure.getRoutineName()));
+    }
+
+    @Override
+    public Optional<ViewExpression> getRowFilter(ConnectorSecurityContext context, SchemaTableName tableName)
+    {
+        throw new IllegalStateException("Not yet supported");
```

Comment:
I think it can be supported when `accessControl` returns `Optional.empty()`

Reply:
When `accessControl` returns a emptyList it can be `Optional.empty()`